enterprise/api/v1/views/default_enterprise_enrollments.py

```diff
@@ -129,8 +129,10 @@ def learner_status(self, request):
             )
 
         # Retrieve configured default enrollment intentions for the enterprise customer
-        default_enrollment_intentions_for_customer = models.DefaultEnterpriseEnrollmentIntention.objects.filter(
-            enterprise_customer=enterprise_customer_uuid,
+        default_enrollment_intentions_for_customer = (
+            models.DefaultEnterpriseEnrollmentIntention.available_objects.filter(
```

Comment:
Breadcrumb for future readers on `SoftDeletableModel`, TIL!
> Note that relying on the default objects manager to filter out not-deleted instances is deprecated. objects will include deleted objects in a future release. Until then, the recommended course of action is to use the manager all_objects when you want to include all instances.
https://django-model-utils.readthedocs.io/en/latest/models.html#softdeletablemodel

Reply:
Yeah, I think the future change to always need to rely on `available_objects` was inspired by this [issue thread](https://github.com/jazzband/django-model-utils/issues/364).